pkg/window/window.go

```diff
@@ -0,0 +1,6 @@
+package window
+
+const (
```

Comment:
💡 This could be too a config passed at the moment of executing the game or in a future configuration page.
So, in the main.go file you could pass a config here:

```go
cfg := window.NewConfig()
sdl.CreateWindowAndRenderer(cfg.Width, cfg.Height, sdl.WINDOW_SHOWN)
```
And the config could be an struct loading the variables from some storage or env:

```go
package window

type Config struct {
  Width int `yaml:"width"`
  Height int `yaml:"height"`
}
```